packages/neon/neon_dashboard/lib/src/pages/main.dart

```diff
@@ -53,4 +83,111 @@ class DashboardMainPage extends StatelessWidget {
       },
     );
   }
+
+  @visibleForTesting
+  // ignore: public_member_api_docs
+  static Iterable<SizedBox> buildWidgetItems({
+    required final BuildContext context,
+    required final dashboard.Widget widget,
+    required final dashboard.WidgetItems? items,
+  }) =>
+      _buildWidgetItems(
+        context: context,
+        widget: widget,
+        items: items,
+      );
```

Comment:
Why not expose the generator function?

Reply:
It's not really needed other than internally and for testing, but I guess to keep the code cleaner it is a good idea :+1: